server.js

```diff
@@ -1,7 +1,39 @@
-const express = require('express')
-const app = express()
-const PORT = 5000
+const express = require('express');
+const mongoose = require('mongoose');
+require('dotenv').config();
+const app = express();
+const PORT = 5000;
+const cors = require('cors');
```

Comment:
Could we have these imports in the right order, I think is better if we keep some kind of ordered logic, right now `cors` is at the end, even though is an import. Do you understand? What do you think?

Reply:
true, i changed it .